src/app/home-page/recent-events/recent-events.component.html

```diff
@@ -31,10 +40,19 @@
             <mat-divider *ngIf="!last" class="custom-margin"></mat-divider>
           </div>
           <div *ngSwitchCase="EventType.ADDTOCOLLECTION">
-            <strong>{{ event.initiatorUser.username }} </strong> added the {{ event?.tool ? 'tool' : 'workflow' }}
-            <a [routerLink]="event?.tool ? '/tools/' + event?.tool.tool_path : '/workflows/' + event?.workflow.full_workflow_path">{{
-              (event?.tool | entryToDisplayName) || (event?.workflow | entryToDisplayName)
-            }}</a>
+            <strong>{{ event.initiatorUser.username }} </strong> added the {{ event?.tool || event?.apptool ? 'tool' : 'workflow' }}
+            <a
```

Comment:
Could avoid duplication with a component or pipe. But that can wait for 1.13.

Reply:
https://ucsc-cgl.atlassian.net/browse/SEAB-4056